eox_tagging/validators.py

```diff
@@ -0,0 +1,295 @@
+        """
+        field_value = getattr(self.instance, field)
+
+        if field_value not in values:
+            # Values allowed is list of values (at least one)
+
+            raise ValidationError("EOX_TAGGING  |   The field {} is not in tag definitions".format(field))
+
+    def validate_exist(self, field, value):  # pylint: disable=unused-argument
+        """
+        Function that validates that the field exists, this means, is not None.
+
+        Arguments:
+            - field: field to validate
+            - value: validations defined for the field
+        """
+
+        field_value = getattr(self.instance, field)
+
+        if not field_value:
+            raise ValidationError("EOX_TAGGING  |   The field {} must exist in the instance created.".format(field))
+
+    def validate_equals(self, field, value):
+        """
+        Function that validates that the field_value is equal to value.
+
+        Arguments:
+            - field: field to validate
+            - value: validations defined for the field
+        """
+        field_value = getattr(self.instance, field)
+
+        field_value = getattr(field_value, "name", field_value)  # In case is choicefield
+
+        if field_value != value:
+            raise ValidationError("EOX_TAGGING  |   The field {} must be equal to {}".format(field, value))
+
+    def validate_object(self, field, value):
+        """
+        Function that validates that the field is instance of value.
+
+        Arguments:
+            - field: field to validate
+            - value: validations defined for the field
+        """
+
+        field_value = getattr(self.instance, "{}_type".format(field))
+
+        if field_value != value:
+            raise ValidationError("EOX_TAGGING  |   The field {} must be an instance of {}.".format(field, value))
+
+    def validate_regex(self, field, value):
+        """
+        Function that validates that the field matches value.
+
+        Arguments:
+            - field: field to validate
+            - value: validations defined for the field
+        """
+
+        field_value = getattr(self.instance, field)
+
+        if not re.search(value, field_value):
+            # Values allowed is regex pattern
+
+            raise ValidationError("EOX_TAGGING  |   The field {} is not in tag definitions".format(field))
+
+    def validate_unique_together(self):
+        """Function that validates that at least one of the two fields in unique together is not null."""
+        for field_tuple in self.instance._meta.unique_together[:]:  # pylint: disable=protected-access
+            if all(getattr(self.instance, field_name) is None for field_name in field_tuple):
+                raise ValidationError("At least one of {field_one} and {field_two} must be not null"
+                                      .format(field_one=field_tuple[0], field_two=field_tuple[1]))
+
+    # Helper functions
+    def __set_defaults(self):
```

Comment:
@felipemontoya I didn't implement this because I have a question, where should we call it?